src/rpcwallet.cpp

```diff
@@ -899,8 +909,38 @@ UniValue sendmany(const UniValue& params, bool fHelp)
 
     EnsureWalletIsUnlocked();
 
-    // Check funds
-    int64_t nBalance = GetAccountBalance(strAccount, nMinDepth);
+    // Check funds & Support non-account sendmany
+    int64_t nBalance;
+
+    if (bFromAccount)
+        nBalance = GetAccountBalance(strAccount, nMinDepth);
```

Comment:
nBalance is now uninitialized. What should it be here?

Reply:
Ugh, how did I miss this. Yes, should be 0.